db/tests/testthat/test.contents_sanity.R

```diff
@@ -6,7 +6,7 @@
 # which accompanies this distribution, and is available at
 # http://opensource.ncsa.illinois.edu/license.html
 #-------------------------------------------------------------------------------
-con <- db.open(list(driver = "PostgreSQL", user = "bety", dbname = "bety", password = "bety"))
+con <- db.open(list(host="psql-pecan.bu.edu", driver = "PostgreSQL", user = "bety", dbname = "bety", password = "bety"))
```

Comment:
May want to reject this change (and next 3, which are equivalent) if it really makes more sense for tests to only pass if you have a local betyDB. But I needed to set the server for the tests to pass, and it seems like other users would too, typically.